santiao-guillen/homework-3/initializeMyName.js

```diff
@@ -0,0 +1,13 @@
+function initializeNames(name) {
+    
+    let array = name.split(' ');
+    for (let i = 1; i < array.length - 1; i++)
```

Comment:
@santiagoGuillenGlob , use modern approaches for array operations like forEach, map, reduce.